src/Ormolu/Printer/Combinators.hs

```diff
@@ -270,3 +277,28 @@ comma = txt ","
 
 space :: R ()
 space = txt " "
+
+----------------------------------------------------------------------------
+-- Layout
+
+data LayoutOptions
+  = LayoutOptions
+      { loSit :: Bool
+      }
+
+defaultLayoutOptions :: LayoutOptions
+defaultLayoutOptions = LayoutOptions False
+
+p_layout :: LayoutOptions -> [R ()] -> R ()
```

Comment:
We should choose a different name for this function because the `p_` convention is reserved for functions printing various parts of AST, it's not for combinators. Perhaps I never documented this explicitly...

Reply:
You are right. I will change it to `layout`.